dagql/idtui/frontend_pretty.go

```diff
@@ -205,6 +225,11 @@ func (fe *frontendPretty) finalRender() error {
 	fe.recalculateViewLocked()
 
 	if fe.Debug || fe.Verbosity >= ShowCompletedVerbosity || fe.err != nil {
+		var msg string
+		if fe.msgPreFinalRender.Len() > 0 {
+			fmt.Fprintln(os.Stderr, fe.msgPreFinalRender.String())
+		}
+		fmt.Fprintln(os.Stderr, msg)
```

Comment:
nit: this results in an extra line at the start of output if `NOTHANKS=1`:

![image](https://github.com/dagger/dagger/assets/7352848/e5784cb1-0151-46e4-96e2-3d69be640f07)

I think we can move this into the condition before?